packages/terra-button-group/tests/wdio/button-group-spec.js

```diff
@@ -97,41 +74,54 @@ describe('Button Group', () => {
       expect(browser.getText('#selected-keys')).to.equal('1');
     });
 
-    Terra.it.isAccessible();
-    Terra.it.matchesScreenshot('button1-selected');
+    Terra.it.validatesElement('button1-selected');
 
     it('should unselect the first button', () => {
       browser.keys('Space');
       expect(browser.getText('#selected-keys')).to.equal('');
     });
 
-    Terra.it.isAccessible();
-    Terra.it.matchesScreenshot('button1-unselected');
+    Terra.it.validatesElement('button1-unselected');
 
     it('should select the second button', () => {
       browser.keys('Tab');
       browser.keys('Space');
       expect(browser.getText('#selected-keys')).to.equal('2');
     });
 
-    Terra.it.isAccessible();
-    Terra.it.matchesScreenshot('button2-selected');
+    Terra.it.validatesElement('button2-selected');
 
     it('should select the third button', () => {
       browser.keys('Tab');
       browser.keys('Space');
       expect(browser.getText('#selected-keys')).to.equal('2, 3');
     });
 
-    Terra.it.isAccessible();
-    Terra.it.matchesScreenshot('button3-selected');
+    Terra.it.validatesElement('button3-selected');
 
     it('should unselect the third button', () => {
       browser.keys('Space');
       expect(browser.getText('#selected-keys')).to.equal('2');
     });
 
-    Terra.it.isAccessible();
-    Terra.it.matchesScreenshot('button3-unselected');
+    Terra.it.validatesElement('button3-unselected');
+  });
+});
+
+Terra.describeViewports('Button Group Wrapping', ['tiny', 'huge'], () => {
```

Comment:
To reduce the number of viewport changes during the test run, we will want to simplify this:

```
Terra.describeViewports('Button Group Wrapping', ['tiny', 'huge'], () => {
    describe('Button Group Wrapping', () => {
         // the test
    });

    describe('Button Group Block', () => {
         // the test
    });
});
```

Reply:
Changed in 57c132c360d0472bd6b073954b90e0144db53946